runtime/v1/shim/client/client.go

```diff
@@ -67,22 +68,24 @@ func WithStart(binary, address, daemonAddress, cgroup string, debug bool, exitHa
 		}
 		defer f.Close()
 
-		var stdoutLog io.ReadWriteCloser
-		var stderrLog io.ReadWriteCloser
-		if debug {
-			stdoutLog, err = v1.OpenShimStdoutLog(ctx, config.WorkDir)
-			if err != nil {
-				return nil, nil, errors.Wrapf(err, "failed to create stdout log")
-			}
-
-			stderrLog, err = v1.OpenShimStderrLog(ctx, config.WorkDir)
-			if err != nil {
-				return nil, nil, errors.Wrapf(err, "failed to create stderr log")
-			}
+		stdoutCopy := ioutil.Discard
+		stderrCopy := ioutil.Discard
+		stdoutLog, err := v1.OpenShimStdoutLog(ctx, config.WorkDir)
+		if err != nil {
+			return nil, nil, errors.Wrapf(err, "failed to create stdout log")
+		}
 
-			go io.Copy(os.Stdout, stdoutLog)
-			go io.Copy(os.Stderr, stderrLog)
+		stderrLog, err := v1.OpenShimStderrLog(ctx, config.WorkDir)
+		if err != nil {
+			return nil, nil, errors.Wrapf(err, "failed to create stderr log")
 		}
+		if debug {
```

Comment:
I think this is pointless. We are already copying, and there can be some errors/warnings from the shim that would be good to have.

Reply:
I agree, I feel like this was an intentional decision made in the past though. Let's see if there was a reason this wasn't done before.

Reply:
But it is what `loadTasks` is doing (Called on containerd restart)

Reply:
@dmcgowan's comment wasn't there when I posted mine... just to disambiguate, my 2nd comment is intended as a follow-up to my first comment.

Reply:
I checked with @crosbymichael on this one as well, he wasn't aware of a reason it is this way. I think we should be OK to just make the behavior the same as `loadTasks` and include that change here.

Reply:
@dmcgowan I don't understand what change needs to be made.
The PR brings this in line with `loadTasks`, but thinking it just doesn't make sense to send to /dev/null in non-debug mode since we are already incurring the cost of creating the pipes and copying over.